Libs/DICOM/Core/ctkDICOMDatabase.cpp

```diff
@@ -659,6 +659,66 @@ QHash<QString,QString> ctkDICOMDatabase::descriptionsForFile(QString fileName)
 }
 
 //------------------------------------------------------------------------------
+QString ctkDICOMDatabase::descriptionForSeries(const QString seriesUID)
+{
+  Q_D(ctkDICOMDatabase);
+
+  QString result;
+
+  QSqlQuery query(d->Database);
+  query.prepare ( "SELECT SeriesDescription FROM Series WHERE SeriesInstanceUID= ?" );
+  query.bindValue ( 0, seriesUID);
+  query.exec();
+
+  if (query.next())
+    {
+    result = query.value(0).toString();
+    }
+
+  return result;
+}
+
+//------------------------------------------------------------------------------
+QString ctkDICOMDatabase::descriptionForStudy(const QString studyUID)
+{
+  Q_D(ctkDICOMDatabase);
+
+  QString result;
+
+  QSqlQuery query(d->Database);
+
+  query.prepare ( "SELECT StudyDescription FROM Studies WHERE StudyInstanceUID= ?" );
+  query.bindValue ( 0, studyUID);
+  query.exec();
+  if (query.next())
+    {
+    result =  query.value(0).toString();
```

Comment:
Right, I had different spacing used in each descriptionForX method, new commit fixes this.